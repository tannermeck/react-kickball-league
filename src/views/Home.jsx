function Home(){
    return (
        <h1>Welcome to Kickball League</h1>
    )
}
export default Home;