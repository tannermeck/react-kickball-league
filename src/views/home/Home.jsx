import './home.css';

function Home(){
    return (
        <h1 className='homeHeader'>Welcome to Kickball League</h1>
    )
}
export default Home;