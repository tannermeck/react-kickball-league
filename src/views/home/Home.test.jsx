import { screen, render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
import Home from './Home';

it('should create a home component', () => {
    const { container } = render(
                            <MemoryRouter>
                                <Home />
                            </MemoryRouter>)
    const label = screen.getByText('Welcome to Kickball League', {exact: false})
    
    expect(label).toBeInTheDocument();
    expect(container).toMatchSnapshot();
})