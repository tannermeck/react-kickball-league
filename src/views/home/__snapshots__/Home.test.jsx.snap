// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should create a home component 1`] = `
<div>
  <h1
    class="homeHeader"
  >
    Welcome to Kickball League
  </h1>
</div>
`;
